docs/tutorial/remark-plugin-tutorial.md

```diff
@@ -264,6 +264,40 @@ module.exports = ({ markdownAST }, pluginOptions) => {
 
 A small library [mdast-util-to-string](https://github.com/syntax-tree/mdast-util-to-string) by Unified was used to extract the plain text of the inner nodes. This would remove links or other types of nodes inside the heading, but given you have full access to the markdown AST, you can modify it however you wish.
 
+## Using asynchronous behavior
+
+Gatsby supports the usage of asynchronous behavior in plugins, and gatsby-transformer-remark`uses [unified](https://github.com/unifiedjs/unified) under the hood, so in this case the`gatsby-remark-purple-headers`transformer can be converted to asynchronous by adding the`async` keyword to the function declaration.
```

Comment:
```suggestion
Gatsby supports the usage of asynchronous behavior in plugins, and `gatsby-transformer-remark` uses [unified](https://github.com/unifiedjs/unified) under the hood. The following example shows how the `gatsby-remark-purple-headers` transformer can be converted to asynchronous by adding the`async` keyword to the function declaration.
```

Reply:
@laurieontech I committed these changes along with the addition of the markdown code title in ec410d6. Thanks!